rhel8/profiles/ospp.profile

```diff
@@ -653,8 +653,8 @@ selections:
     #echo "# you MAY comment out the next line." >> $CONFIG
     #echo "user.max_user_namespaces = 0" >> $CONFIG
 
-    ## TO DO: https://github.com/ComplianceAsCode/content/issues/4493
-    #echo "kernel.unprivileged_bpf_disabled = 1" >> $CONFIG
+    ## RHEL 8 CCE-82203-1: Disable Access to Network bpf() Syscall From Unprivileged Processes
+    - sysctl_kernel_unprivileged_bpf_disabled
```

Comment:
The rule selection is already covered in #4575, please remove it from here to avoid conflicts.